test/README.md

```diff
@@ -168,3 +182,20 @@ Windows vs non-windows portability env vars:
     SEP:           \ or / (required)
     OBJ:          .obj or .o (required)
     EXE:          .exe or <null> (required)
+
+BASH Tests
+------------------------------
```

Comment:
Some markdown engines require a newline here (and you might want to align the length of the bar with heading or use `##` if you don't care)